test/fail_compilation/test21164.d

```diff
@@ -0,0 +1,13 @@
+/* REQUIRED_ARGS: -Ifail_compilation
+TEST_OUTPUT:
+---
+fail_compilation/test21164d.d(3): Error: (expression) expected following `static if`
+fail_compilation/test21164d.d(3): Error: found `}` instead of statement
+fail_compilation/test21164.d(11): Error: template instance `test21164a.D!(R!(O(), 1))` error instantiating
+---
+*/
+import test21164a;
```

Comment:
```suggestion
import imports.test21164a;
```

Reply:
doesn't need to happen, I changed the paths.

Reply:
dang that doesn't even work

Reply:
https://github.com/dlang/dmd/pull/11585#discussion_r484523485